runtime/include/tasks/qthreads/tasks-qthreads.h

```diff
@@ -117,6 +117,16 @@ extern
 volatile int chpl_qthread_done_initializing;
 
 typedef struct {
+  //chpl_bool serial_state;
```

Comment:
Actually, fifo doesn't need them. I don't think it'd be wrong to move them to the common code anyway...